src/proposals/mips/mip-b25/MTokens.json

```diff
@@ -0,0 +1,21 @@
+[
+    {
+        "addressesString": "MOONWELL_weETH",
+        "supplyCap": 900e18,
+        "borrowCap": 350e18,
+        "collateralFactor": 0.74e18,
+        "initialMintAmount": 1e12,
+        "name": "Moonwell weETH",
+        "priceFeedName": "CHAINLINK_WEETH_ETH_COMPOSITE_ORACLE",
```

Comment:
Please update this to use a market rate oracle rather than the exchange rate oracle.